### ISSUE TITLE
«Invalid postMessage origin: https://connect.squareup.com»

### ISSUE THREAD

**Issue by dmitrii-fediuk** [mage2pro/square#5]
It does not break anything, but it would be nice to find out the reason of this message.
![04](https://user-images.githubusercontent.com/83204/31193829-e56b6a64-a94d-11e7-9d43-d68e0401fdda.png)

**Comment by dmitrii-fediuk** [comment]
I have asked it in the official forum: https://www.sellercommunity.com/t5/Troubleshooting/Connect-API-v2-Invalid-postMessage-origin-https-connect-squareup/m-p/51854

**Comment by dmitrii-fediuk** [comment]
A similar issue in Firefox (I am unable to reproduce it): [[Square] The payment form fields are not loaded properly in Mozilla Firefox: «**Cannot read property 'postMessage' of null**»](https://mage2.pro/t/topic/4631)

**Comment by ThomasLRoberts** [comment]
It has been several weeks. Any resolution?

**Comment by dmitrii-fediuk** [comment]
@ThomasLRoberts 
Which website have you bought the extension's license for?

**Comment by ThomasLRoberts** [comment]
https://familypettraining.com

**Comment by dmitrii-fediuk** [comment]
@ThomasLRoberts
But your website uses WordPress, not Magento.
My extension is for Magento 2, not for WordPress.

**Comment by ThomasLRoberts** [comment]
yes, but I am having the same difficulty and was hoping that perhaps you had found a solution that might provide insight into my problem

**Comment by dmitrii-fediuk** [comment]
As I have stated [in the message above](https://github.com/mage2pro/square/issues/5#issuecomment-334257446), I [asked](https://www.sellercommunity.com/t5/E-Commerce-API/Connect-API-v2-Invalid-postMessage-origin-https-connect-squareup/m-p/51854) this question in the official Square forum at **2017-10-04**.  
Nobody answers me.
See also: [[Square] How to contact the technical support?](https://mage2.pro/t/2829)
Previously, Square's employees answered my other questions asked in the same way in the official Square forum.

**Comment by ThomasLRoberts** [comment]
I had also asked Square tech support and have no response. Thanks
